cmd/sonobuoy/app/results.go

```diff
@@ -100,6 +100,13 @@ func NewCmdResults() *cobra.Command {
 // getReader returns a *results.Reader along with a cleanup function to close the
 // underlying readers. The cleanup function is guaranteed to never be nil.
 func getReader(filepath string) (*results.Reader, func(), error) {
+	fi, err := os.Stat(filepath)
+	if err != nil {
+		return nil, func() {}, err
+	}
+	if fi.IsDir() {
+		return results.NewReaderFromDir(filepath), func() {}, nil
+	}
```

Comment:
This is almost the only code needed to make `sonobuoy results` work on a directory. Easy win if we already wanted to do the other bits.